PARAMETERS_MAPPING/unit_view.csv

```diff
@@ -0,0 +1,61 @@
+id;name;short_name;definition;date_added
+451;Bytes;Bytes                                                                                                         ;;2013-07-18
+769;Cells per millilitre;Cells ml-1                                                                                                    ;;2014-07-10
+445;Decibars;dBar                                                                                                          ;;2013-07-18
+439;Decibels;dbel                                                                                                          ;;2013-07-18
+440;Degrees;deg                                                                                                           ;;2013-07-18
+429;Degrees Celsius;degC                                                                                                          ;;2013-07-18
+430;Degrees Kelvin;degK                                                                                                          ;;2013-07-18
```

Comment:
Kelvin are not degrees. Just Kelvin.

Reply:
Changed corresponding file accordingly, thanks @mhidas